scripts/data_aggregation_summary_scripts/create_data_provenance_summary.py

```diff
@@ -0,0 +1,131 @@
+
+    write_csv_file(csv_content)
+
+
+def parse_input(argv):
+
+    conp_dataset_dir = None
+
+    description = '\nThis tool facilitates the aggregation of data provenance for reporting purposes.' \
+                  ' It will read DATS files and print out a summary of data provenance based on the `origin`' \
+                  ' fields present in the DATS.json files of every dataset present in conp-dataset directory.\n'
+    usage = (
+        '\n'
+        'usage  : python ' + __file__ + ' -d <conp-dataset directory path>\n\n'
+        'options: \n'
+            '\t-d: path to the conp-dataset directory to parse\n'
+    )
+
+    try:
+        opts, args = getopt.getopt(argv, "hd:")
+    except getopt.GetoptError:
+        sys.exit()
+
+    for opt, arg in opts:
+        if opt == '-h':
+            print(description + usage)
+            sys.exit()
+        elif opt == '-d':
+            conp_dataset_dir = arg
+
+    if not conp_dataset_dir:
+        print('a path to the conp-dataset needs to be given as an argument to the script by using the option `-d`')
+        print(description + usage)
+        sys.exit()
+
+    if not os.path.exists(conp_dataset_dir + '/projects'):
+        print(conp_dataset_dir + 'does not appear to be a valid path and does not include a `projects` directory')
+        print(description + usage)
+        sys.exit()
+
+    return conp_dataset_dir
+
+
+def read_conp_dataset_dir(conp_dataset_dir):
+
+    dataset_dirs_list = os.listdir(conp_dataset_dir + '/projects')
+
+    csv_content = [
+        ['Dataset', 'Principal Investigator', 'Consortium', 'Institution', 'City', 'Province', 'Country']
+    ]
+
+    for dataset in dataset_dirs_list:
+        if dataset == '.touchfile':
+            continue
+        dats_path = conp_dataset_dir + '/projects/' + dataset + '/DATS.json'
+        if not (os.path.exists(dats_path)):
+            subdataset_content_list = look_for_dats_file_in_subfolders(conp_dataset_dir, dataset)
+            csv_content.extend(subdataset_content_list)
+            continue
+
+        parsed_result = parse_dats_json_file(dats_path)
+        csv_content.append(parsed_result)
+
+    return csv_content
+
+
+def look_for_dats_file_in_subfolders(conp_dataset_dir, dataset):
+
+    subdataset_dirs_list = os.listdir(conp_dataset_dir + '/projects/' + dataset)
+
+    subdataset_content = []
+
+    for subdataset in subdataset_dirs_list:
+        dats_path = conp_dataset_dir + '/projects/' + dataset + '/' + subdataset + '/DATS.json'
+        parsed_result = parse_dats_json_file(dats_path)
+        subdataset_content.append(parsed_result)
+
+    return subdataset_content
+
+
+def parse_dats_json_file(dats_path):
+
+    with open(dats_path) as dats_file:
+        dats_dict = json.loads(dats_file.read())
+
+    extra_properties = dats_dict['extraProperties']
+    values_dict = {}
+    for extra_property in extra_properties:
+        values_dict[extra_property['category']] = extra_property['values'][0]['value']
```

Comment:
Now it only takes first value in the list. We can have use cases where `'origin_institution'` contains more than one institution,
can be fixed with
`values_dict[extra_property['category']] = ", ".join(str(value) for value in [exp["value"] for exp in extra_property["values"]])`